src/commands/modifyMultisig.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command, flags } from '@oclif/command';
+import { BootstrapService, BootstrapUtils } from '../service';
+import { AnnounceService } from '../service/AnnounceService';
+import { CommandUtils } from '../service/CommandUtils';
+
+export default class ModifyMultisig extends Command {
+    static description = `Create or modify a multisig account`;
+
+    static examples = [
+        `$ symbol-bootstrap modifyMultisig`,
+        `$ echo "$MY_ENV_VAR_PASSWORD" | symbol-bootstrap modifyMultisig --useKnownRestGateways`,
+    ];
+
+    static flags = {
+        help: CommandUtils.helpFlag,
+        target: CommandUtils.targetFlag,
+        minRemovalDelta: flags.integer({
+            description:
+                'Delta of signatures needed to remove a cosignatory. ' +
+                '0 means no change, a positive(+) number means increment and a negative(-) number means decrement to the actual value.',
+            char: 'r',
+        }),
+        minApprovalDelta: flags.integer({
+            description:
+                'Delta of signatures needed to approve a transaction. ' +
+                '0 means no change, a positive(+) number means increment and a negative(-) number means decrement to the actual value.',
+            char: 'a',
+        }),
+        addressAdditions: flags.string({
+            description: 'Cosignatory accounts addresses to be added (separated by a comma).',
```

Comment:
have you tried `multiple: true`, cli handles lists of strings. Does it work nicely?

Reply:
Yes, tried multiple addresses separated by comma, works fine 👍

Reply:
I mean, oclif `multiple: true` field in the flags.string. It users cli style multiple values

Reply:
ignore, --addressAdditions A,B,C its easier than --addressAdditions A --addressAdditions B --addressAdditions C

does  `--addressAdditions` (empty) work? empty list no prompt.

Reply:
I think so
>ignore, --addressAdditions A,B,C is easier than --addressAdditions A --addressAdditions B --addressAdditions C

No it doesn't, it gives => No Error: Flag --addressAdditions expects a value
> does --addressAdditions (empty) work? empty list no prompt.

If we need to support no-prompt command as well, I think we have 2 options here:
1. Remove all the prompts, use only the flag values
2. Add the following flags --no-addressAdditions, --no-addressDeletions 
    and the command will look like
    `symbol-bootstrap modifyMultisig --no-addressAdditions --no-addressDeletions --minApprovalDelta=1 --minRemovalDelta=0`
I'd probably vote for 2.

Reply:
Updated the code as it will not prompt for the following.
`--addressAdditions='' --addressDeletions='' `